.github/workflows/test.yml

```diff
@@ -0,0 +1,25 @@
+name: test
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [12.x, 14.x]
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node-version }}
+    - run: npm ci
+    - run: npm test
```

Comment:
Do we want to run `standard --fix` in this action as well? The fastify-opentelemetry [validate action](https://github.com/autotelic/fastify-opentelemetry/blob/master/.github/workflows/validate.yml#L15) runs a `validate` script which runs both of the `fix` and `test` [scripts](https://github.com/autotelic/fastify-opentelemetry/blob/master/package.json#L7-L12).  

Should I set up something similar in fastify-hmac?

Reply:
The --fix flag won't be useful on CI (we've already committed and pushed at this point). We could lint as well if we want but it would be up to the developer to resolve the failing build locally, fix the changes and push them.

Reply:
RIght. I was misreading the validate command. Do want me to set this branch up to include linting in the action or should we just merge it and deal with it later?

Reply:
Good call, may as well add linting here too in case something gets by our precommit. Updated 👍